pkg/resources/schema_acceptance_test.go

```diff
@@ -470,27 +470,39 @@ func TestAcc_Schema_ManagePublic(t *testing.T) {
 				ExpectError: regexp.MustCompile("Error: error creating schema PUBLIC"),
 			},
 			{
-				ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
-				ConfigDirectory:          acc.ConfigurationDirectory("TestAcc_Schema/basic_with_pipe_execution_paused"),
-				ConfigVariables: map[string]config.Variable{
-					"name":                  config.StringVariable(name),
-					"database":              config.StringVariable(acc.TestDatabaseName),
-					"pipe_execution_paused": config.BoolVariable(true),
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"snowflake": {
+						VersionConstraint: "=0.94.0",
+						Source:            "Snowflake-Labs/snowflake",
+					},
 				},
+				Config: schemav094WithPipeExecutionPaused(name, acc.TestDatabaseName, true),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("snowflake_schema.test", "name", name),
 					resource.TestCheckResourceAttr("snowflake_schema.test", "database", acc.TestDatabaseName),
 					resource.TestCheckResourceAttr("snowflake_schema.test", "pipe_execution_paused", "true"),
 				),
 			},
 			{
-				ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
-				ConfigDirectory:          acc.ConfigurationDirectory("TestAcc_Schema/basic_with_pipe_execution_paused"),
-				ConfigVariables: map[string]config.Variable{
-					"name":                  config.StringVariable(name),
-					"database":              config.StringVariable(acc.TestDatabaseName),
-					"pipe_execution_paused": config.BoolVariable(false),
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"snowflake": {
+						VersionConstraint: "=0.94.0",
+						Source:            "Snowflake-Labs/snowflake",
+					},
 				},
+				PreConfig: func() {
+					// In v0.94 `CREATE OR REPLACE` was called, so we should see a DROP event.
+					schemas := acc.TestClient().Schema.ShowWithOptions(t, &sdk.ShowSchemaOptions{
+						History: sdk.Pointer(true),
+						Like: &sdk.Like{
+							Pattern: sdk.String(schemaId.Name()),
+						},
+					})
+					require.Len(t, schemas, 2)
+					require.Zero(t, schemas[0].DroppedOn)
+					require.NotZero(t, schemas[1].DroppedOn)
```

Comment:
nit: Not ideal, because we rely on Snowflake's output ordering

Reply:
Added sorting.